js/index.hbs

```diff
@@ -30,7 +31,12 @@
         </nav>
 
         <div class="StyleguideContent">
-            <iframe class="StyleguideExample" name="StyleguideExample" src="about:blank"></iframe>
+          <div class="StyleguideTabs-wrapper">
```

Comment:
Could this wrapper be removed so that the opening element/block of the tabs has a class of `StyleguideTabs`?